src/details/ArborX_Ray.hpp

```diff
@@ -201,7 +207,10 @@ KOKKOS_INLINE_FUNCTION bool solveQuadratic(float const a, float const b,
 
   auto const discriminant = b * b - 4 * a * c;
   if (discriminant < 0)
+  {
+    x1 = x2 = 0 / 0.f;  // quiet NaN
```

Comment:
not sure about this here. Is it necessary?

Reply:
I changed it.  It is now handled in `intersection(Ray, Box, float&, float&) instead` and I elected to set `tmin = INF` and `tmax = -INF` instead.